client-firewall-rules.md

```diff
@@ -3,38 +3,110 @@
+- [Data Endpoint Change](#data-endpoint-change)
+- [Registry FQDN Endpoints](#registry-fqdn-endpoints)
+- [Testing the `*.cdn.mscr.io` Data Endpoint](#testing-the-cdnmscrio-data-endpoint)
+- [Testing the `*.data.mcr.microsoft.com` Data Endpoint](#testing-the-datamcrmicrosoftcom-data-endpoint)
+- [Q&A](#qa)
+- [Rollout Status](#rollout-status)
 
 ## Configuring Client Firewall Rules
 
+> ### **DATA ENDPOINT CHANGE**
+> **Note: to provide users enough time to adjust, we have extended the change date to June 15, 2020.**  
+> To provide a consistent FQDN between the REST and data endpoints, beginning **June 15, 2020** the data endpoint will change from `*.cdn.mscr.io` to `*.data.mcr.microsoft.com`  
+Deployments follow [Azure Safe Deployment Practices][azure-safe-deployment], which means changes will rollout, per region over several days. For this change, we will take a slower pace, rolling out a few regions and taking a pause of a few days to monitor feedback.  
+Please monitor this page for [updates](#rollout-status), including the regional rollout dates and when the rollout is complete. 
 
 
-> ### **DATA ENDPOINT CHANGE**
-> To provide a consistent FQDN between the REST and data endpoints, beginning ~~March 3,~~ **April 13 2020** the data endpoint will be changing from `*.cdn.mscr.io` to `*.data.mcr.microsoft.com`  
-Deployments follow [Azure Safe Deployment Practices][azure-safe-deployment], which means changes will rollout, per region, over several days.  
-Once a change of the data endpoint is confirmed, clients can remove the `*.cdn.mscr.io` FQDN.  
-**March 9th Update:**  
-During the multi-region deployment, we discovered AKS node scaling and service updates could fail. To minimize the impact, we are rolling back the data endpoint change which will complete Friday, March 13, 2020.  
-**New Deployment:** To assure all MCR customers have consistent and trusted domains between the registry and data endpoints, we will start a new `*.data.mcr.microsoft.com` data endpoint deployment **Monday April 13, 2020**.
+## Registry FQDN Endpoints
 
 To access MCR, the following FQDNs are required. 
 
 | Purpose | Protocol | Target FQDN | Available |
 | - | - | - | - |
 | Registry Endpoint | https | `mcr.microsoft.com` | Now |
-| Data Endpoint | https | `*.cdn.mscr.io` | Now - April 20, 2020 |
-| Data Endpoint | https | `*.data.mcr.microsoft.com` | Starting April  13, 2020 |
+| Data Endpoint | https | `*.cdn.mscr.io` | Now |
+| Data Endpoint | https | `*.data.mcr.microsoft.com` | Starting June  15, 2020 |
 
-If you're updating this prior to April 13, adding both date endpoints will assure a smooth transition. Once the consolidated data endpoint deployment is complete, we'll update this page.
+If you're updating this prior to June 15th, adding both date endpoints will assure a smooth transition. Once the consolidated data endpoint deployment is complete, we'll update this page enabling the removal of `*.cdn.mscr.io`.
```

Comment:
You have a spelling error: "adding both date endpoints" should read "adding both data endpoints"